### PR TITLE
Create bug issue form

### PR DESCRIPTION
**Pull request by sven1103**
Simplify bug reporting with forms.

### CODE REVIEW

**Comment by sven1103** [comment]
Requires #755 first, so we can reference the code of conduct.

File: .github/ISSUE_TEMPLATE/bug [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://example.com).
```
Comment (Steffengreiner) [inline .github/ISSUE_TEMPLATE/bug:43]: Don't forget to link the code of conduct here once the PR is merged for it :)

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
I compared it with the Github [documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) and have 3 questions: 

1.) Shouldn't this be a bug.yml file? 
2.) Can the previous markdown bug_report.md file then be removed? 
3.) Shouldn't we also provide a config.yml file to ensure no blank issues can be created? See [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser)

**Review by Steffengreiner — DISMISSED** [review]
Let's see how it looks 🚀

**Review by Steffengreiner — APPROVED** [review]
Ship it 🚢

**Pull request merged**